ci/ci-test-tasks/push-nuget-init.yml

```diff
@@ -0,0 +1,29 @@
+# requires some rework - relies on batch processing and steps are not self-evindent
```

Comment:
All the nuget functionality is also redundant since this pipeline should never publish to nuget.

Reply:
Got it